### PR TITLE
PR on behalf of @infiniteverma: Format Slideshows Part 1/3

### PR DESCRIPTION
**Pull request by alodahl**
solves #598 .  I have not heard from this person in weeks, but it seems finished, and I think they should get the points.

Mentors, please choose 5 slideshows at random  in #598  and make sure they follow the format mentioned.  If it doesn't, please comment about it.  Slides 30 is an exception, because neither @InfiniteVerma nor I could get edit rights to it.

### CODE REVIEW

File: learning-to-learn/learning-to-learn.md [hunk 1]
```diff
@@ -87,4 +87,7 @@ Take the [Growth Mindset Self-Assessment](http://mindsetonline.com/testyourminds
 
 ### Check for Understanding
 
-Participants write a self-reflective journal entry covering: 1) What they learned about themselves, 2) What from this lesson they felt challeneged by, 3) What from this lesson they want to practice.
+Participants write a self-reflective journal entry covering: 
+1. What they learned about themselves
+1. What from this lesson they felt challeneged by
```
Comment (manaswinidas) [inline learning-to-learn/learning-to-learn.md]: Looks good! A small typo in challenged

Comment (alodahl) [reply learning-to-learn/learning-to-learn.md]: We are not actually editing this file; we are checking the slideshows, @manaswinidas

Comment (alodahl) [reply learning-to-learn/learning-to-learn.md]: @manaswinidas @hoduchieu01 @Kundan28 please be sure to read the PR descriptions

**Comment by Kundan28** [comment]
@alodahl I am not having edit access to the sides. Please give access for slideshow no. 25, 26, 27, 28 & 29.

**Comment by alodahl** [comment]
@kundan28 you should have comment rights to 25, 26, 28, and 29. 27 is the example, and I made it, so I hope it’s right! Pick a different one for your 5th choice.

**Comment by Kundan28** [comment]
> @Kundan28 you should have comment rights to 25, 26, 28, and 29. 27 is the example, and I made it, so I hope it’s right! Pick a different one for your 5th choice.

I will choose 24.

**Comment by alodahl** [comment]
> > @Kundan28 you should have comment rights to 25, 26, 28, and 29. 27 is the example, and I made it, so I hope it’s right! Pick a different one for your 5th choice.
> 
> I will choose 24.

We’re you able to look at those yet?

**Comment by Kundan28** [comment]
> We’re you able to look at those yet?

By just having comments rights I can't review the formatting of the slides.

**Comment by alodahl** [comment]
@Kundan28 sorry, I didn't realized you couldnt see formatting. I added you to 24-29.

**Comment by Kundan28** [comment]
> @Kundan28 sorry, I didn't realized you couldnt see formatting. I added you to 24-29.

No problem, I will review them by today EOD.

**Comment by Kundan28** [comment]
> @Kundan28 sorry, I didn't realized you couldnt see formatting. I added you to 24-29.

@alodahl I have reviewed and commented the changes to be made in the slideshow 29, 28 & 24.
And I don't have edit access for slideshow 25 & 26.

**Comment by alodahl** [comment]
@kundan28 thank you for looking at those... it looks like we may need to put this back in to-do.  @hoduchieu01 @manaswini-das did you notice any that were perfect that I should take out of the issue?

**Comment by Kundan28** [comment]
> @Kundan28 thank you for looking at those... it looks like we may need to put this back in to-do.

Yeah, I also think so.

**Comment by alodahl** [comment]
this is going back to To-Do.

**Pull request closed without merging**